### PR TITLE
Added sponsors to the website. See below for more important changes:

### PR DESCRIPTION
**Pull request by SmithcS**
- Changed the .section-header, .header-text, and .header-bar so that there is one main declaration at the top, and then section specific css happens in each sections own scope
- Added some more section borders that were needed to transition between sponsor and footer and faq to sponsor sections
- Added SVGS for sponsors. Sponsor logos that are wider than they are tall  250x500px and everything else is 500x500px

Responsiveness is 👍

### CODE REVIEW

**Comment by SmithcS** [comment]
@andrewltran39 @angelachan64 Please check this out when you get a chance! This will help make sense of how we'll add more sponsors in the future. I'd like to talk to you guys about improving the look of the section as well - right now it is all white to make sure that a sponsor's logo will show up regardless of color but there is plenty off cool things that we can do moving forward. I also really really want to add some graphics down there. ~ So many possibilities ~

**Comment by sushshring** [comment]
I'm a bit concerned about the repeated white section. Would it be possible to generate some graphic so that the colors don't get super repeated? Either some other graphics or by making the about section flow into a different color.

File: website/index.html [hunk 1]
```diff
@@ -214,19 +215,18 @@
         </div>
       </div>
     </section>
+
     <div class="section-separator">
       <img class="section-separator-background" src="assets/images/border2-whitebg.svg">
     </div>
+
```
Comment (sushshring) [inline website/index.html]: nitpick: whitespace change.

File: website/scss/main.scss [hunk 2]
```diff
@@ -421,6 +412,10 @@ header {
   padding-top:      50px;
   background-color: #194E6B;
 
+  .header-bar {
```
Comment (sushshring) [inline website/scss/main.scss]: Repeated selector?

**Comment by andrewltran39** [comment]
sponsor section is looking good

My suggestion for the background color is to keep it white during the daytime (depending on timezone) with the sponsor logos blue as default then the original logo color when hovered (same features from the last hackathon)
During nighttime, have the background color turn to the nightblue with the logo colors being white/lightgray

**Comment by SmithcS** [comment]
@andrewltran39 I'm down for that, maybe we do move back to white logos...
@sushshring @Ryguy11o @angelachan64, thoughts on moving back to white logos? Would make the day/night theme a lot easier to deal with

File: website/index.html [hunk 3]
```diff
@@ -494,9 +494,50 @@
...
+          <div class="section-header col-xs-12">
+            <div class="header-text">Sponsors</div>
+            <div class="header-bar"></div>
+          </div>
+        </div>
+
+        <!-- Silver Sponsors -->
+        <div class="row silver">
+          <div class="offset-sm-3 col-sm-6">
+            <a href="https://www.kcftech.com/">
```
Comment (sushshring) [inline website/index.html]: Add `target=_blank`. Add to other sponsor logos as well.

**Comment by sushshring** [comment]
Let's make this the PR for the newer graphics, since @Ryguy11o already added in the necessary sponsors for now.

**Comment by sushshring** [comment]
Closing since way out of date.

**Pull request closed without merging**